### ISSUE TITLE
Pretrain model converge to one value

### ISSUE THREAD

**Issue by Tianyangg** [MrGiovanni/ModelsGenesis#25]
Hello,

First thanks for your amazing work. I am training my own genesis network Using lung CT scans. The patches generated includes a reasonable amount of Lung part while when training with Genesis_Chest_CT, the output of the network will learn one single value for all pixels, any idea why this happened?

Thanks,

Tian

![image](https://user-images.githubusercontent.com/39927555/87907020-564ccb80-ca96-11ea-803a-b223f1d0b4b3.png)

**Comment by akiliyiu** [comment]
I have the same problem here. Did you find the solution?

**Comment by MrGiovanni** [comment]
Hi @akiliyiu and @Tianyangg 

I need more details to resolve this issue. Would you please send me an email so that we can arrange a time to look into your implementation?

Thanks,
Zongwei

**Comment by emergencyd** [comment]
I have the same problem. The prediction converges to all 0, while the loss stays around 0.096. There is something that is different from common settings.
1) It seems that the default learning rate is 1. Is this too large?
2) The only data augmentation is flip. But for medical images (bone CT for my case), is this reasonable? Since default arrangement is necessary for feature learning. We usually do little flip and rotation.

@akiliyiu @Tianyangg @MrGiovanni Thank you!
